src/views/Pages/LoginPage.jsx

```diff
@@ -44,13 +47,40 @@ class LoginPage extends React.Component {
     clearTimeout(this.timeOutFunction);
     this.timeOutFunction = null;
   }
+
+  onHandleLogin = (event) => {
+    event.preventDefault();
+
+    let username = event.target.firstname.value;
+    let password = event.target.password.value;
+
+    const data = {
+      username, password
+    };
+
+    this.props.UserActions.loginUserAction(data);
+  }
+
   render() {
     const { classes } = this.props;
+
+    let isSuccess, message;
+
+    if (this.props.response.login.hasOwnProperty('response')) {
+      isSuccess = this.props.response.login.response.success;
```

Comment:
if this variables will only be inside this scope i dont see the need of declaring them before as let variables, so declare it as as a  const in this line and remove de #67